src/tasks/nli_probing.py

```diff
@@ -7,123 +7,95 @@
-                            s1_idx=8, s2_idx=9, label_idx=10, label_fn=label_fn, skip_rows=1,
+       
+        prob_data = load_tsv(data_file=os.path.join(path, ), max_seq_len=max_seq_len,
+                            s1_idx=8, s2_idx=9, label_idx=10, label_fn=targ_map.__getitem__, skip_rows=1,
                             tokenizer_name=self._tokenizer_name)
-        te_data = load_tsv(data_file=os.path.join(path, 'test_dummy.tsv'), max_seq_len=max_seq_len,
-                           s1_idx=1, s2_idx=2, label_idx=None, skip_rows=0,
-                           tokenizer_name=self._tokenizer_name)
-
-        self.train_data_text = tr_data
-        self.val_data_text = val_data
-        self.test_data_text = te_data
+        
+        self.train_data_text = self.val_data_text = self.test_data_text = prob_data
         log.info("\tFinished loading negation data.")
 
 
 @register_task('nli-prob-prepswap', rel_path='NLI-Prob/')
 class NLITypeProbingTaskPrepswap(PairClassificationTask):
 
-    def __init__(self, path, max_seq_len, name, probe_path="probe_dummy.tsv",
-                 **kw):
+    def __init__(self, path, max_seq_len, name, **kw):
         super(NLITypeProbingTaskPrepswap, self).__init__(name, n_classes=3,
                                                          **kw)
-        self.load_data(path, max_seq_len, probe_path)
+        self.load_data(path, max_seq_len)
         self.sentences = self.train_data_text[0] + self.train_data_text[1] + \
             self.val_data_text[0] + self.val_data_text[1]
 
-    def load_data(self, path, max_seq_len, probe_path):
-        tr_data = load_tsv(data_file=os.path.join(path, 'train_dummy.tsv'), max_seq_len=max_seq_len,
-                           s1_idx=1, s2_idx=2, label_idx=None, skip_rows=0,
-                           tokenizer_name=self._tokenizer_name)
-        # TODO(?): should this use probe_path?
-        val_data = load_tsv(data_file=os.path.join(path, 'all.prepswap.turk.newlabels.tsv'),
+    def load_data(self, path, max_seq_len):
+        prob_data = load_tsv(data_file=os.path.join(path, 'all.prepswap.turk.newlabels.tsv'),
                             max_seq_len=max_seq_len, s1_idx=8, s2_idx=9, label_idx=0, skip_rows=0,
                             tokenizer_name=self._tokenizer_name)
-        te_data = load_tsv(data_file=os.path.join(path, 'test_dummy.tsv'), max_seq_len=max_seq_len,
-                           s1_idx=1, s2_idx=2, label_idx=None, skip_rows=0,
-                           tokenizer_name=self._tokenizer_name)
 
-        self.train_data_text = tr_data
-        self.val_data_text = val_data
-        self.test_data_text = te_data
+        self.train_data_text = self.val_data_text = self.test_data_text = prob_data
         log.info("\tFinished loading preposition swap data.")
 
 
 @register_task('nli-alt', rel_path='NLI-Prob/')
-class NLITypeProbingAltTask(NLITypeProbingTask):
```

Comment:
The author declared this to be a subclass of NLITypeProbingTask, but it didn't really inherit anything. Nor is this subclass relationship used anywhere where (like in an if-clause) in jiant. 
So I thought, better keep the conformity.